### PR TITLE
[footer] telegram integration

### PR DESCRIPTION
**Pull request by gyurka00**
Add telegram link into the footer.

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit d8b3347a8e0f257d57dde63aa8aca91a774f24ff

https://app.netlify.com/sites/augmint/deploys/5ab0e1f748769b1c6027a305

File: src/containers/app/AppFooter.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import Subscribe from "./Subscribe";
 import backgroundImg from "assets/images/globe.png";
 import discordLogo from "assets/images/Discord-Logo.svg";
 import decent from 'assets/images/decent.png';
+import telegramLogo from 'assets/images/telegram_logo.png';
```
Comment (reyrodrigez) [inline src/containers/app/AppFooter.js]: how about switching to svg? https://commons.wikimedia.org/wiki/File:Telegram_logo.svg

File: src/containers/app/AppFooter.js [hunk 2]
```diff
@@ -29,18 +30,31 @@ export function AppFooter(props) {
...
+                    <Segment
+                        basic
+                        className="join-telegram"
+                        size="tiny"
+                        as="a"
+                        href="https://t.me/augmint"
+                        target="_blank"
+                    >
+                        <Image src={telegramLogo} />
+                        Talk to us on Telegram
```
Comment (reyrodrigez) [inline src/containers/app/AppFooter.js]: can we say `Talk with us on Telegram`?

File: src/containers/app/AppFooter.js [hunk 3]
```diff
@@ -29,18 +30,31 @@ export function AppFooter(props) {
...
+                    <Segment
+                        basic
+                        className="join-discord"
+                        size="tiny"
+                        as="a"
+                        href="https://discord.gg/PwDmsnu"
+                        target="_blank"
+                    >
+                        <Image src={discordLogo} />
+                        Talk to us on Discord
```
Comment (reyrodrigez) [inline src/containers/app/AppFooter.js]: can we say `Talk with us on Discord`?

File: src/containers/app/AppFooter.js (hunk 1 above)
Comment (gyurka00) [reply src/containers/app/AppFooter.js]: Thanks. I fixed it.

File: src/containers/app/AppFooter.js (hunk 2 above)
Comment (gyurka00) [reply src/containers/app/AppFooter.js]: @szerintedmi told me, we talk about it later

File: src/containers/app/AppFooter.js (hunk 3 above)
Comment (gyurka00) [reply src/containers/app/AppFooter.js]: @szerintedmi told me, we talk about it later

**Pull request merged**